src/api/json/catalog.json

```diff
@@ -3096,6 +3096,12 @@
       "fileMatch": ["*discovery.yml"],
       "url": "https://raw.githubusercontent.com/meltano/meltano/main/src/meltano/schemas/discovery.schema.json"
     },
+    {
+      "name": "Metadata for a Bazel module",
+      "description": "Metadata file for a Bazel module",
+      "fileMatch": ["metadata.json"],
```

Comment:
This is a generic file match pattern. Can you make it more specific by specifying what folder the file should be in as part of the pattern?

Reply:
Found a more concise filematch